### PR TITLE
Adding ROF to MCH simulation digits

### PR DESCRIPTION
**Pull request by mwinn2**

### CODE REVIEW

File: Detectors/MUON/MCH/Workflow/src/digits-reader-workflow.cxx [hunk 1]
```diff
@@ -15,33 +15,12 @@
 /// \author Andrea Ferrero, CEA
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Workflow/src/digits-reader-workflow.cxx]: I believe this file should not be modified by this PR, right ?

File: Detectors/MUON/MCH/Workflow/src/DigitReaderSpec.cxx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+#include "Framework/DataSpecUtils.h"
+#include "Framework/Logger.h"
+#include "Framework/Output.h"
+#include "Framework/Task.h"
+#include "Framework/WorkflowSpec.h"
+#include "SimulationDataFormat/MCTruthContainer.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+#include "DataFormatsMCH/Digit.h"
+#include "DataFormatsMCH/ROFRecord.h"
+//#include "MIDSimulation/ColumnDataMC.h"
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Workflow/src/DigitReaderSpec.cxx]: please remove commented out code

File: Detectors/MUON/MCH/Workflow/src/digits-reader-workflow.cxx (hunk 1 above)
Comment (mwinn2) [reply Detectors/MUON/MCH/Workflow/src/digits-reader-workflow.cxx]: @aphecetche : no it shouldn't
I will now update a version with the file from dev.

File: Detectors/MUON/MCH/Workflow/src/DigitReaderSpec.cxx (hunk 2 above)
Comment (mwinn2) [reply Detectors/MUON/MCH/Workflow/src/DigitReaderSpec.cxx]: Ok done

File: Detectors/MUON/MCH/Workflow/src/digits-reader-workflow.cxx (hunk 1 above)
Comment (aphecetche) [reply Detectors/MUON/MCH/Workflow/src/digits-reader-workflow.cxx]: @mwinn2 best would be to simply exclude it from this PR altogether as it's not supposed to be changed, right ?

Comment (mwinn2) [reply Detectors/MUON/MCH/Workflow/src/digits-reader-workflow.cxx]: @aphecetche Ok.

**Review by aphecetche — APPROVED** [review]

**Comment by aphecetche** [comment]
@shahor02 or @sawenzel could you please have a look and/or merge please ? that might not be our last word on the matter, but at least it should allow us to start making full chain simulation tests with MCH.

**Comment by mwinn2** [comment]
@sawenzel : Hi Sandro, did you have time to look into the pull request?

**Pull request merged**